apps/question-the-docs/backend/app.py

```diff
@@ -23,12 +37,11 @@ def startup_db_client():
         # We wrap our MongoDB to make it a SuperDuperDB!
         _app.superduperdb = superduper(_app.mongodb)
 
-        # EXPLAIN ARTIFACTS HERE.
+        # Artifacts are data that has been pre-processed for AI.
         load_ai_artifacts(_app.superduperdb)
 
-        # We populate our SuperDuperDB with AI components.
-        # EXPLAIN COMPONENTS HERE.
-        # These will be used later to answer questions on our data.
+        # Components are AI models that have been selected based
```

Comment:
Hmm, expressing it that way is, I think, misleading - it sounds like there's some sort of dispatch mechanism that selects things.

"A Component is an AI Model.  

Love it so far.

"Different types of Components can process different types of Artifacts."

a bit subpar... what about:

"A Component can process one or more types of Artifact."

That's clear and short!

Is that true?  Or is it "exactly one type"?

---


"A Component is an AI Model.  Each Component can process one or more types of Artifact."